### DISCUSSION TITLE
Sql Profiler auto capturing events

### DISCUSSION THREAD

**Discussion by SharmaAntriksh** [DaxStudio/DaxStudio#1156] [General]
I just noticed that if you launch SQL Profiler from DAX Studio, it already is capturing some events. I remember this wasn't the case with DS2, is this supposed be like this? I have no issues with it, I can always close it and start a new trace but thought of highlighting since I never saw this behaviour earlier.

**Comment by dgosbell** [comment]
If this has changed it would be a change in Profiler, not DAX Studio. I only pass a connection details to SQL Profiler as command line arguments, I don't think it's possible to tell it whether or not to start tracing and I cannot see an option to do this.

**Reply by SharmaAntriksh** [reply]
Thanks!
